### ISSUE TITLE
While using the TOC with the Avada theme and page builder, the TOC was showing twice.

### ISSUE THREAD

**Issue by Sdaud1950** [ahmedkaludi/Easy-Table-of-Contents#432]
When we add the TOC with the Avada theme with Fusion Core, then the TOC showing double in the set position works well, but on the footer, the TOC is showing.

Ref ticket: https://magazine3.in/conversation/112900?folder_id=29

**Comment by Sanjeevsetu** [comment]
This is not right solution. We have to find better one so moving it on next milestone.

**Comment by muslimakhan** [comment]
Footer is not showing after this above solution: 
https://wordpress.org/support/topic/footer-is-not-displayed-after-update-2-0-47-with-avada/

 So comment on this code, and will be resolved this ticket in future.
